notebook/automl_classification.ipynb

```diff
@@ -113,7 +113,8 @@
     "settings = {\n",
     "    \"time_budget\": 600,  # total running time in seconds\n",
     "    \"metric\": 'accuracy',  # can be: 'r2', 'rmse', 'mae', 'mse', 'accuracy', 'roc_auc', 'roc_auc_ovr',\n",
-    "                           # 'roc_auc_ovo', 'log_loss', 'mape', 'f1', 'ap', 'ndcg', 'micro_f1', 'macro_f1'\n",
+    "                        # 'roc_auc_ovo', 'log_loss', 'mape', 'f1', 'ap', 'ndcg', 'micro_f1', 'macro_f1'\n",
+    "                        # check the documentation for full list of arguments(https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML/)\n",
```

Comment:
Remove "can be...". Change the URL to be https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#optimization-metric